<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VIP Club</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-11</y>
      <width>1280</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBoat">
    <property name="title">
     <string>Boat</string>
    </property>
    <addaction name="actionAddBoat"/>
    <addaction name="actionListBoat"/>
   </widget>
   <widget class="QMenu" name="menuMitglied">
    <property name="title">
     <string>Mitglied</string>
    </property>
    <addaction name="actionNewMitglied"/>
    <addaction name="actionListMitglied"/>
   </widget>
   <addaction name="menuMitglied"/>
   <addaction name="menuBoat"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewMitglied">
   <property name="icon">
    <iconset theme="list-add"/>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionListMitglied">
   <property name="icon">
    <iconset theme="format-justify-fill"/>
   </property>
   <property name="text">
    <string>List</string>
   </property>
  </action>
  <action name="actionAddBoat">
   <property name="icon">
    <iconset theme="list-add"/>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionListBoat">
   <property name="icon">
    <iconset theme="format-justify-fill"/>
   </property>
   <property name="text">
    <string>List</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
